myfitnesspal/client.py

```diff
@@ -220,6 +232,55 @@ def get_date(self, *args, **kwargs):
 
         return day
 
+    def get_measurements(self, measurement='Weight', start=None, end=None):
+        if start is None:
```

Comment:
Please don't take this as a suggestion that you change anything here, but I wonder if it might make more sense if `start` and `end` were reversed here?  I don't really use this functionality, but it seems like it *might* be confusing that `start` is expected to be later than `end`.